app/graphql/index.js

```diff
@@ -1,30 +1,64 @@
 const { makeExecutableSchema } = require('graphql-tools'),
   { applyMiddleware } = require('graphql-middleware'),
-  types = require('./types'),
-  inputs = require('./inputs'),
-  users = require('./users'),
-  healthCheck = require('./healthCheck');
+  fs = require('fs'),
+  path = require('path');
 
-const typeDefs = [types, inputs, ...users.schemas, ...healthCheck.schemas];
+const importEverything = () => {
+  const imports = { modules: [] };
+  fs.readdirSync(__dirname, { withFileTypes: true }).forEach(dirent => {
+    if (dirent.isDirectory()) {
+      imports.modules.push({ [dirent.name]: require(path.join(__dirname, dirent.name)) });
+    } else if (dirent.name === 'index.js') {
+      return;
+    }
+    imports[dirent.name.replace(/.js/gi, '')] = require(path.join(__dirname, dirent.name));
+  });
+  return imports;
+};
+
+const { types, inputs, enums: Enums, modules } = importEverything();
+
+const getSchemas = () =>
```

Comment:
getSchemas, getResolvers and getMiddlewares have duplicated logic. Can we abstract the common behaviour?

Reply:
Jmm the thing is I don't see how. Because although they have very similar bodies, they differ in some crucial parts... So I don't really know how to abstract it. I accept suggestions hehehe

Reply:
mm can we do something like getKeysAndReturnElements?